src/components/_hero.scss

```diff
@@ -1,7 +1,7 @@
 // Extension to the Design System hero component
 .hero {
   // sass-lint:disable no-duplicate-properties
-  margin-bottom: rem($spacing-large);
+  margin-bottom: rem($nds-spacing-large);
 
   // Hack to 'break out' of the container (layout) for hero components
```

Comment:
Didn't yout put the 'break out' CSS directly in the design system in v1, in which case we can remove this break out?